lsr_role2collection/collection_readme.txt

```diff
@@ -0,0 +1,25 @@
+# Fedora Linux System Roles Ansible Collection
```

Comment:
I don't think we want to use "Fedora" in this file.  I think we should use "Linux System Roles Ansible Collection" as the name, and use "linux" instead of "Fedora" below.  We should not mention Red Hat Customer Portal or Automation Hub.  There are some other copy/paste issues.
I do realize that makes it much harder to use this file as the source for the downstream readme, which should have "Red Hat" and "RHEL" in it.